koku/masu/processor/orchestrator.py

```diff
@@ -296,7 +296,7 @@ def start_manifest_processing(  # noqa: C901
                     )
                     continue
 
-                cache_key = f"{provider_uuid}:{report_file}"
+                cache_key = f"{provider_uuid}:{report_file}"  # noqa: E231
```

Comment:
Same question, why is this change needed?

Reply:
same as above.